pom.xml

```diff
@@ -197,18 +197,41 @@
                </environments>
             </configuration>
     </plugin>
-	<plugin>
-		<groupId>org.eclipse.tycho</groupId>
-		<artifactId>tycho-surefire-plugin</artifactId>
-		<version>${tycho.version}</version>
-		<configuration>
-			<useUIHarness>true</useUIHarness>
-			<useUIThread>false</useUIThread>
-			<product>org.eclipse.platform.ide</product>
-		</configuration>
-	</plugin>
-
-      </plugins>
+    <plugin>
+	    <groupId>org.eclipse.tycho</groupId>
+	    <artifactId>tycho-surefire-plugin</artifactId>
+	    <version>${tycho.version}</version>
+	    <configuration>
+		    <useUIHarness>true</useUIHarness>
+		    <useUIThread>false</useUIThread>
+		    <product>org.eclipse.platform.ide</product>
+		    <skipTests>true</skipTests>
+	    </configuration>
+    </plugin>
+    <plugin>
+	    <groupId>org.jacoco</groupId>
+	    <artifactId>jacoco-maven-plugin</artifactId>
+	    <version>0.8.7</version>
+	    <executions>
+		    <execution>
+			    <goals>
+				    <goal>prepare-agent</goal>
+		            </goals>
+		    </execution>
+		    <execution>
+			    <id>report</id>
+		            <phase>verify</phase>
+			    <goals>
+				    <goal>report-aggregate</goal>
+		            </goals>
+			    <configuration>
+				    <skip>true</skip>
```

Comment:
@wolfgang-ch By default, I have set it so that the code coverage is not run because it takes more time for the build to complete (I ran it on my machine and it took  21 mins instead of 4). This way, when you run the build on your machine or for the new MT releases, it takes the same time as before. It is only activated for the builds running on GitHub.

Reply:
I think to be more climate friendly, this process should run on github not more than once a day and not after each main commit or each PR

Reply:
Great idea ! I will set that up. Do you have a preference for what time the daily build should run ?

Reply:
> 
> 
> Great idea ! I will set that up. Do you have a preference for what time the daily build should run ?

see https://github.com/wolfgang-ch/mytourbook/pull/415#discussion_r739960638